R/tanglegram.R

```diff
@@ -488,7 +488,9 @@ plot_horiz.dendrogram <- function(x,
     } else {
       segments(x0, hgt, x0, yTop)
     }
+    print("x")
```

Comment:
Hey, did you mean to keep these prints?
They seem redundant.

Reply:
Not at all, good catch